Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/initial-letter/index.md

### Values

> The keyword value `normal` or a `<number>` optionally followed by an `<integer>`:

Reviewer: ```suggestion
This property is specified as `normal` or a `<number>` optionally followed by an `<integer>`:
```
